### DISCUSSION TITLE
Export/Import

### DISCUSSION THREAD

**Discussion by ShootyGroove** [phase1geo/Minder#438] [Q&A]
First I created a map of a database structure that I am working on. After that I cannot remember if I used the export function or just grabbed the .minder file but when I transferred the file to another machine and opened it, all of the text in the file is now gone. The map structure is still there, connectors, style, layout, etc but the text is gone. The worst part is I cannot remember everything that went into these areas! Help please

Any help would be much appreciated.

![minderFile](https://user-images.githubusercontent.com/88412158/145389216-8fd9540e-56e9-473a-8541-11b7e07597f9.png)

**Comment by phase1geo** [comment]
Could you email me the .minder file on the second computer?

**Reply by phase1geo** [reply]
Which version of Minder was used to create the file and which version was used to open the file?

**Comment by ShootyGroove** [comment]
Yes, the version I am running is 1.9.2-1

minder file should be attached.

Thank you!
Mike
